src/core/abr/representation_estimator.ts

```diff
@@ -191,6 +192,9 @@ export default function RepresentationEstimator({
                                                                        initialBitrate,
                                               lowLatencyMode);
   const requestsStore = new PendingRequestsStore();
+  let mayBeFromCache = generateCachedSegmentFinder();
+
+  let hasLoadedNonCachedSegments = false;
```

Comment:
If you went all the way to create that "finder" why not just find a solution to integrate that variable to it?